apps/meteor/server/models/raw/MatrixBridgedRoom.ts

```diff
@@ -0,0 +1,39 @@
+import type { IMatrixBridgedRoom, RocketChatRecordDeleted } from '@rocket.chat/core-typings';
+import type { IMatrixBridgedRoomModel } from '@rocket.chat/model-typings';
+import type { Collection, Db, IndexDescription } from 'mongodb';
+
+import { BaseRaw } from './BaseRaw';
+
+export class MatrixBridgedRoomRaw extends BaseRaw<IMatrixBridgedRoom> implements IMatrixBridgedRoomModel {
+	constructor(db: Db, trash?: Collection<RocketChatRecordDeleted<IMatrixBridgedRoom>>) {
+		super(db, 'matrix_bridged_rooms', trash);
+	}
+
+	protected modelIndexes(): IndexDescription[] {
+		return [
+			{ key: { rid: 1, mri: 1 }, unique: true, sparse: true },
```

Comment:
there is no need for this index since there is no specific `find` using both fields at the same time
```suggestion
```